fullscreen/api/element-request-fullscreen-svg-svg-manual.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<title>Element.requestFullscreen() for SVG svg element</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../trusted-click.js"></script>
+<div id="log"></div>
+<svg></svg>
+<script>
+async_test(function(t)
+{
+    var svg = document.querySelector("svg");
+    assert_true(svg instanceof SVGSVGElement);
+    trusted_request(svg, document.body);
+    document.onfullscreenchange = t.step_func_done();
```

Comment:
I think this test can only call `t.step_func_done()` once, where would the second time be?

In testharness.js, nothing that happens after the first failure or done event matters, so in general it's fine if there are two ways the test can pass and if both happen.